avocado/core/jobdata.py

```diff
@@ -41,6 +43,20 @@
 CMDLINE_FILENAME = 'cmdline'
 
 
+def _find_class(module, name):
+    """
+    Look for a class including compatibility workarounds
+    """
+    if module == "avocado.plugins.yaml_to_mux":
+        mod = __import__("avocado_varianter_yaml_to_mux", fromlist=[module])
+        return getattr(mod, name)
+    else:
+        mod = __import__(module)
+        mod = sys.modules[module]
+        klass = getattr(mod, name)
+        return klass
```

Comment:
Yep, it's a remnant from development. I'll return the getattr in v2. What do you mean by the `level=0` restriction? The original method on my system looks like this:

```python
    def find_class(self, module, name):
        # Subclasses may override this
        __import__(module)
        mod = sys.modules[module]
        klass = getattr(mod, name)
        return klass
```